src/components/tables/Captures/Rows.tsx

```diff
@@ -0,0 +1,30 @@
+import { TableCell, TableRow } from '@mui/material';
+import { FormattedDate } from 'react-intl';
+
+interface Props {
+    discovers: any[];
+    styling: any;
+}
+
+function Rows({ discovers, styling }: Props) {
```

Comment:
Same as the empty one... not sure how I feel about the pattern but kind of like it. Only thing that sucks is the disconnect between the rows and columns specified in the parent.